src/main/java/io/swagger/codegen/v3/generators/typescript/TypeScriptAngularClientCodegen.java

```diff
@@ -119,7 +119,7 @@ public void processOpts() {
         if (additionalProperties.containsKey(NG_VERSION)) {
             ngVersion = new SemVer(additionalProperties.get(NG_VERSION).toString());
         } else {
-            ngVersion = new SemVer("4.3.0");
+            ngVersion = new SemVer("6.0.0");
```

Comment:
@dennisameling Please note that the help still states that "Default is '4.3'". Probably should extract that magic string to a constant and fix that.